policies/main.rego

```diff
@@ -9,3 +9,9 @@ deny = {denial |
 skip(test_name) {
 	data.config.policy.non_blocking_checks[_] == test_name
 }
+
+skip(policy_name) {
+	# Use the nanosecond epoch defined in the policy if present. Otherwise, use now.
+	when_ns := object.get(data.config.policy, ["when_ns"], time.now_ns())
```

Comment:
TIL from the [docs](https://www.openpolicyagent.org/docs/latest/policy-reference/#time):

> Multiple calls to the `time.now_ns` built-in function within a single policy evaluation query will always return the same value.

I thought there might be a subtle race condition here but that is not the case.